hack/olm-registry/olm-artifacts-template.yaml

```
@@ -16,6 +16,8 @@ parameters:
   value: '["None"]'
 - name: DEADMANSSNITCH_OSD_TAGS
   required: true
+- name: FEDRAMP
+  value: "${FEDRAMP}"
```

This needs to default to "false".
```suggestion
  value: "false"
```